### ISSUE TITLE
Undefined index: instance in .../Plugin/Event/Invoker.php on line 53

### ISSUE THREAD

**Issue by mamsincl** [Smile-SA/magento2-module-debug-toolbar#7]
Notice: Undefined index: instance in /var/www/html/vendor/smile/module-debug-toolbar/Plugin/Event/Invoker.php on line 53

after mining the issue has been caused by an observer has been disabled in adminhtml scope using way have been declared by Magento itself (https://devdocs.magento.com/guides/v2.3/extension-dev-guide/events-and-observers.html#disabling-an-observer) 
The observer was declared in default scope initialy

**Comment by guvra** [comment]
Hi,

I guess we need to add the same check as the Magento Invoker:

```php
public function aroundDispatch(...)
{
    if (isset($configuration['disabled']) && $configuration['disabled'] === true) {
        return $closure($configuration, $observer);
    }

    ...
}
```

**Comment by guvra** [comment]
Solved in commit eb65a02f3e1af634aa2efa30133b4da6902d1106
